opflexagent/test/test_async_port_manager.py

```diff
@@ -191,7 +191,7 @@ def test_apply_config_fails(self):
         self.manager.schedule_update(to_schedule)
         update = self._get_device_requests(['1', '2', '4'])
         self.manager._opflex_endpoint_update(mock.Mock(), update)
-        self.assertRaises(Exception, self.manager.apply_config)
+        self.assertRaises(Exception, self.manager.apply_config)  # noqa
```

Comment:
Why is noqa needed here? Seems odd.

Reply:
Pep8 was saying that Exception is "too general", but I don't know enough about the code to know of a more specific exception to expect.

Reply:
It's passing the exception above - you could just use something like ValueError in both places, and you could get rid of the noqa.

Reply:
ValueError worked - thanks!